api/valor_api/schemas/metrics.py

```diff
@@ -565,8 +592,10 @@ def db_mapping(self, evaluation_id: int) -> dict:
         ----------
         A mapping dictionary.
         """
+        value = self.value if self.value is not None else -1
```

Comment:
can we combine these two if statements so that we aren't assigning to `value` twice?